docs/pages/components/SearchField.mdx

```diff
@@ -35,6 +35,14 @@ import { SearchField } from '@deque/cauldron-react';
 </FieldWrap>
 ```
 
+### Error
+
+```jsx example
+<FieldWrap>
+  <SearchField label="Error" error="This field has an error" />
+</FieldWrap>
+```
+
```

Comment:
I think @scurker mentioned to me in a DM that we don't need to handle error states for this component (since searches don't usually error)

Reply:
Oh jk, looks like you already commented

Reply:
@dequejosie is there a need for an error state in what you're implementing in the extension?